ci-operator/step-registry/rh-ecosystem-edge/assisted-mcp/install/rh-ecosystem-edge-assisted-mcp-install-commands.sh

```diff
@@ -16,4 +16,18 @@ if [[ -d "assisted-service-mcp" ]]; then
     fi
 fi
 
+PWD=$(pwd)
+
+if [[ "$(basename $PWD)" != "assisted-service-mcp" ]]; then
+    git clone https://github.com/openshift-assisted/assisted-service-mcp.git
+    cd assisted-service-mcp
+    git fetch origin
+    git checkout $ASSISTED_MCP_GIT_BRANCH
```

Comment:
Isn't this the same as what we do above?

Could we move the conditional and clone above the existing block on line 9 and deduplicate this?